nautobot/core/views/generic.py

```diff
@@ -255,17 +254,17 @@ def get(self, request):
                 self.queryset = self.queryset.none()
 
             display_filter_params = [
-                check_filter_for_display(filterset_filters, field_name, values)
+                check_filter_for_display(filterset.filters, field_name, values)
                 for field_name, values in filter_params.items()
             ]
 
             if request.GET:
                 factory_formset_params = convert_querydict_to_factory_formset_acceptable_querydict(
-                    request.GET, self.filterset
+                    request.GET, filterset
                 )
-                dynamic_filter_form = DynamicFilterFormSet(filterset_class=self.filterset, data=factory_formset_params)
+                dynamic_filter_form = DynamicFilterFormSet(filterset_class=filterset, data=factory_formset_params)
```

Comment:
Can we rename the `filterset_class` param to `filterset` to reflect its change in usage, or is that an API breakage?